features/step_definitions/journey_test_steps.rb

```diff
@@ -0,0 +1,339 @@
+require "ostruct"
+require "rest_client"
+
+store = OpenStruct.new
```

Comment:
Alexey did some work to remove the need for a global store and to use instance variables instead.  It would be good to do the same thing here.